frontend/src/components/common/table/table.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactElement } from 'react';
+import { Column, useTable } from 'react-table';
+import styles from './table/styles.module.scss';
+
+type Props<Data extends Record<string, unknown>> = {
+  columns: Column<Data>[];
+  data: readonly Data[];
+};
+const Table = <Data extends Record<string, unknown>>({
+  columns,
+  data,
+}: Props<Data>): ReactElement => {
+  const tableInstance = useTable({
+    columns: columns as Column<Record<string, unknown>>[],
+    data: data as Record<string, unknown>[],
+  });
```

Comment:
We don't need this more
```suggestion
  const tableInstance = useTable({
    columns,
    data,
  });
```